### DISCUSSION TITLE
acteur wit

### DISCUSSION THREAD

**Discussion by ABINolan** [witsa/synapps#770] [Q&A] [answered]
Bonjour,

Ma question concerne un acteur dans le synoptique, je sais qu'il est facile de piloter une consigne par exemple sur Synapps en activant son reflet depuis l'hôte. Cependant savez-vous si il est possible de la commander directement depuis les synoptiques de wit.

Merci

**Comment by JeremieDENAEGEL** [comment, marked as answer]
Bonjour,
Les acteurs synoptiques de base "objet" et "variable d'une ressource" couplés à un acteur synoptique "bouton" pour faire le BP de validation permettent de modifier des données.
Il existe d'autres méthodes suivant le visuel souhaité.
